sdk/core/azure-core/src/samples/java/com/azure/core/util/tracing/TracerJavaDocCodeSnippets.java

```diff
@@ -36,22 +38,27 @@ public void startTracingSpan() {
 
         // start a new tracing span with explicit parent, sets the request attributes on the span and sets the span
         // kind to client when process kind SEND
-        tracer.start("azure.eventhubs.send", sendContext, ProcessKind.SEND);
+        Context updatedSendContext = tracer.start("azure.eventhubs.send", sendContext, ProcessKind.SEND);
+        System.out.printf("Span returned in the context object: %s%n",
+            updatedSendContext.getData(OPENCENSUS_SPAN_KEY).get());
         // END: com.azure.core.util.tracing.start#string-context-processKind-SEND
 
         // BEGIN: com.azure.core.util.tracing.start#string-context-processKind-RECEIVE
         // start a new tracing span with explicit parent, sets the diagnostic Id (traceparent headers) on the current
         // context when process kind RECEIVE
-        Context updatedContext = tracer.start("azure.eventhubs.receive", traceContext, ProcessKind.RECEIVE);
-        System.out.printf("Diagnostic Id: {} %s%n", (String) updatedContext.getData(DIAGNOSTIC_ID_KEY).get());
+        Context updatedReceiveContext = tracer.start("azure.eventhubs.receive", traceContext,
+            ProcessKind.RECEIVE);
+        System.out.printf("Diagnostic Id: {} %s%n", (String) updatedReceiveContext.getData(DIAGNOSTIC_ID_KEY).get());
         // END: com.azure.core.util.tracing.start#string-context-processKind-RECEIVE
 
         // BEGIN: com.azure.core.util.tracing.start#string-context-processKind-PROCESS
         // start a new tracing span with remote parent and uses the span in the current context to return a scope
         // when process kind PROCESS
         Context processContext = new Context(OPENCENSUS_SPAN_KEY, "<user-current-span>")
             .addData(SPAN_CONTEXT, "<user-current-span-context>");
-        tracer.start("azure.eventhubs.process", processContext, ProcessKind.PROCESS);
+        Context updatedProcessContext = tracer.start("azure.eventhubs.process", processContext,
+            ProcessKind.PROCESS);
+        System.out.printf("Scope: {} %s%n", updatedProcessContext.getData("scope").get().getClass());
```

Comment:
{} is not required. Do you need .getClass?

Reply:
I was just doing that to use the returned object and avoid one of our checkstyle issues.
And thought printing object won't return much information, so..